src/uri/params/from_form_data.cr

```diff
@@ -0,0 +1,47 @@
+def Array.from_form_data(params : URI::Params, name : String)
+  params.fetch_all(name).map { |item| T.from_form_data(params, name).as T }
+end
+
+def Bool.from_form_data(params : URI::Params, name : String)
+  case params[name]?
+  when "true", "1", "yes", "on"  then true
+  when "false", "0", "no", "off" then false
+  end
+end
+
+def Number.from_form_data(params : URI::Params, name : String)
+  return nil unless value = params[name]?
+
+  new value, whitespace: false
```

Comment:
Using `whitespace: false` here so that it doesn't consider `age=   25` to be valid.